Bottom: cb5c06f098c709e903d100114aff1ffeebd2d434
Top:    cb5c06f098c709e903d100114aff1ffeebd2d434
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-23 18:21:40 +0000

Try adding polygon objects after each clipping stage. rtree->faster??

*** ACTUALLY SLOWS STARTUP DOWN ***




---


